### PR TITLE
[WIP] Add Get Involved page.

### LINKED ISSUE #423: Get Involved Page
This is in reference to the discussion going on in politeia issue 747: https://github.com/decred/politeiagui/issues/747

I think it would be appropriate to add something to decred.org that contains some information for new users about getting involved and links to some of the existing and future content on the subject, but it shouldn't replace the existing content entirely.

### PR DESCRIPTION
**Pull request by ZeroASIC**
Closes #423

### CODE REVIEW

**Comment by ZeroASIC** [comment]
I included pts6.js even though we're not using those anymore.  It's the file I originally intended to use in the header, but we're using videos now.  Hopefully we can get the correct videos generated from it.  For now I just duplicated the videos from the contributors page.

**Comment by peterzen** [comment]
The [Recruitment](https://blog.decred.org/2017/07/25/Decred-Recruiting/)  blog post moved in house: https://decred.org/recruiting/

File: src/i18n/en.json [hunk 1]
```diff
@@ -822,5 +830,97 @@
   {
     "id": "Back to home page",
     "translation": "Back to home page"
+  },
+  {
+    "id": "getinvolved_description",
+    "translation": "Wondering how to get involved with Decred? We have a whole list of projects for you to contribute to. Get familiar with the project your interested in, hack on it, and send us a pull request!"
```
Comment (RichardRed0x) [inline src/i18n/en.json]:
```suggestion
    "translation": "Wondering how to get involved with Decred? We have a whole list of projects for you to contribute to. Get familiar with the project you're interested in, hack on it, and send us a pull request!"
```

Comment (RichardRed0x) [reply src/i18n/en.json]: Can also add link from "get involved" to new docs page on contributing to Decred, once it's live.

File: src/i18n/en.json [hunk 2]
```diff
@@ -822,5 +830,97 @@
...
+  },
+  {
+    "id": "getinvolved_repos_title",
+    "translation": "Decred related GitHub repositories"
+  },
+  {
+    "id": "getinvolved_repos_text",
+    "translation": "For maintaining a list of GitHub repositories related to the Decred project. If you have a repository to add, open a pull request adding the details."
```
Comment (RichardRed0x) [inline src/i18n/en.json]:
```suggestion
    "translation": "This page showcases a list of GitHub repositories related to the Decred project. If you have a repository to add, open a pull request adding the details."
```

File: src/i18n/en.json [hunk 3]
```diff
@@ -822,5 +830,97 @@
...
+    "id": "getinvolved_notsoftware_desc_dcrdesign",
+    "translation": " "
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_video-production",
+    "translation": "production of Decred video explainers, including scripts and storyboards for work in production. These videos go through stages of production and feedback is typically sought in <a href=\"https://matrix.to/#/#writers:decred.org\">#DCR Writers</a> at certain milestones."
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_decred-news",
+    "translation": "monthly <a href=\"https://xaur.github.io/decred-news/\">Decred Journal</a> maintained by @bee. Seeks review and contributions from other community members in <a href=\"https://matrix.to/#/#documentation:decred.org\">#DCR documentation</a> around the start of each month, with the journal being published around the 6th."
```
Comment (RichardRed0x) [inline src/i18n/en.json]:
```suggestion
    "translation": "monthly <a href=\"https://xaur.github.io/decred-news/\">Decred Journal</a> maintained by @bee. and open for contributions in the latest draft branch. @bee seeks review from other community members in <a href=\"https://matrix.to/#/#documentation:decred.org\">#DCR documentation</a> around the start of each month, with the journal being published around the 6th."
```

File: src/i18n/en.json [hunk 4]
```diff
@@ -822,5 +830,97 @@
...
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_decred-events",
+    "translation": "@vj maintains a set of guidelines and best practices for attending events and hosting meetups. Community members should submit logs of events where Decred had a presence as well as information on upcoming events."
```
Comment (RichardRed0x) [inline src/i18n/en.json]:
```suggestion
    "translation": "@vj maintains a set of guidelines and best practices for attending events and hosting meetups. Community members are invited to submit logs of events where Decred had a presence as well as information on upcoming events."
```

File: src/i18n/en.json [hunk 5]
```diff
@@ -822,5 +830,97 @@
...
+    "id": "getinvolved_notsoftware_desc_dcrstatus",
+    "translation": "dashboard for Decred Command Line Interface users."
+  },
+  {
+    "id": "getinvolved_additional_title",
+    "translation": "Additional Resources"
+  },
+  {
+    "id": "getinvolved_additional_recruiting",
+    "translation": "<a href=\"https://decred.org/recruiting/\">Decred Recruiting</a> - The original post about joining decred."
```
Comment (RichardRed0x) [inline src/i18n/en.json]: This can probably go once we link to new docs "becoming a contributor" page

File: src/i18n/en.json [hunk 6]
```diff
@@ -822,5 +830,97 @@
...
+    "id": "getinvolved_notsoftware_desc_decred-news",
+    "translation": "monthly <a href=\"https://xaur.github.io/decred-news/\">Decred Journal</a> maintained by @bee. Seeks review and contributions from other community members in <a href=\"https://matrix.to/#/#documentation:decred.org\">#DCR documentation</a> around the start of each month, with the journal being published around the 6th."
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_decred-events",
+    "translation": "@vj maintains a set of guidelines and best practices for attending events and hosting meetups. Community members should submit logs of events where Decred had a presence as well as information on upcoming events."
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_richardred0x",
+    "translation": "has repositories for <a href=\"https://github.com/RichardRed0x/writing\">writing</a> (feedback usually sought before publication), <a href=\"https://github.com/RichardRed0x/analysis\">analysis</a> (data and scripts available, welcomes improvements or extensions to code), and an older repo with <a href=\"https://github.com/RichardRed0x/dash-analysis\">Dash data and analysis</a>."
```
Comment (RichardRed0x) [inline src/i18n/en.json]: I would drop this bullet and replace it with new bullets, see comment on the PR for list of suggestions.

**Comment by RichardRed0x** [comment]
I'm sure I missed some repos that would be good additions to this page, if anyone thinks of any add them here.

File: src/i18n/en.json (hunk 6 above)
Comment (ZeroASIC) [reply src/i18n/en.json]: I reworded this a little bit and then added the new bullets underneath it.

File: src/i18n/en.json (hunk 5 above)
Comment (ZeroASIC) [reply src/i18n/en.json]: The new document would be a really good addition to this.  See my comments in #423.

**Comment by linnutee** [comment]
Kudos on efforts @ZeroASIC, however would prefer in future to sync up in slack/matrix before any bigger undertakings. Not preferred workflow to see a crude version get released, which will need to be fixed by others.

**Pull request closed without merging**